Gemfile

```diff
@@ -64,6 +64,17 @@ group :job do
   gem 'sequel', require: false
 end
 
+# Action Cable
+group :cable do
+  gem 'faye-websocket',   '~> 0.10.0', require: false
```

Comment:
We can remove all these dependencies but puma from here. They are all in the gemspec.